src/os/bluestore/bluefs_types.cc

```diff
@@ -152,11 +152,11 @@ bluefs_fnode_delta_t* bluefs_fnode_t::make_delta(bluefs_fnode_delta_t* delta) {
   delta->ino = ino;
   delta->size = size;
   delta->mtime = mtime;
-  delta->offset = allocated_commited;
+  delta->offset = allocated_committed;
```

Comment:
Waffling, but I'm inclined to accept this as it appears to change all uses consistently, if there's an omission it won't compile.  @epuertat ?

Reply:
Yeah, this is no shell script :P so unless all references are updated to the same, it'll fail to compile.